Arrowgene.Ddon.LoginServer/Handler/GetCharacterListHandler.cs

```diff
@@ -39,6 +44,27 @@ public override void Handle(LoginClient client, IPacket packet)
                 cResponse.CharacterListElement.CommunityCharacterBaseInfo.CharacterName.LastName = c.LastName;
                 cResponse.CharacterListElement.CurrentJobBaseInfo.Job = c.Job;
                 cResponse.CharacterListElement.CurrentJobBaseInfo.Level = (byte) c.ActiveCharacterJobData.Lv;
+
+                List<CDataGPCourseValid> ValidCourses = new List<CDataGPCourseValid>();
+
+                foreach (var ValidCourse in _AssetRepo.GPCourseInfoAsset.ValidCourses)
+                {
+                    CDataGPCourseValid cDataGPCourseValid = new CDataGPCourseValid()
+                    {
+                        Id = c.CharacterId,
+                        CourseId = ValidCourse.Value.Id,
+                        NameA = _AssetRepo.GPCourseInfoAsset.Courses[ValidCourse.Value.Id].Name, // Course Name
+                        NameB = "https://members.dd-on.jp/sp_ingame/img/icon/bouken.jpg", // Link to a JPEG
```

Comment:
The server hosts an HTTP server, a server owner could want to add a jpg in the embedded HTTP server and have it show here. This URL should probably come from the same info as the rest of the passport course info

Reply:
I am not sure what the paths to all these images should be. Rumi only had access to Course 1 9 (Adventures Passport) and Course 15 (All courses free of charge) from what I could tell. What should we fill in for the other courses?

Reply:
My idea is that they could be customizable, if the rest of the info is taken from the JSON, then the URL should too come from it, so if a server admin wants to add their own passport course with different effects and different image. This would also allow to easily add the rest of the images to be accurate to the original servers if theyre figured out later down the line